test-requirements.in

```diff
@@ -0,0 +1,11 @@
+pytest == 3.7.4  # for catchlog fixture
```

Comment:
I think this pin is an artifact from recent pytest trouble; and should just be `>= 3.3`.

Reply:
Wow, nice catch! Fixed. I made sure there were no others invalid changes like that in the .in files. Thanks.